pkg/apis/componentconfig/types.go

```diff
@@ -111,6 +111,9 @@ type KubeSchedulerConfiguration struct {
 	// Indicate the "all topologies" set for empty topologyKey when it's used for PreferredDuringScheduling pod anti-affinity.
 	// DEPRECATED: This is no longer used.
 	FailureDomains string
+
+	// CacheDebugResyncInterval is the interval between cache rebuild and compare. It's specified as integer as number of seconds.
+	CacheDebugResyncInterval int32
```

Comment:
Why not make this a metav1.Duration?

Reply:
(you can register duration flags with FlagSet.DurationVar, and target the internal time.Duration in metav1.Duration)

Reply:
I am not aware of this option. I will take a look.